src/main/java/edu/ucla/library/bucketeer/verticles/FinalizeJobVerticle.java

```diff
@@ -67,6 +69,23 @@ protected Logger getLogger() {
         return LOGGER;
     }
 
+    /**
+     * Extract a simple URL, throwing out extra path/query/etc elements.
+     *
+     * @param aLongURL The source URL to be stripped dowwn
+     */
+    private String getSimpleURL(final String aLongURL) {
+        try {
+            final URL url = new URL(aLongURL);
+            final StringBuffer buffer = new StringBuffer().append(url.getProtocol()).append("://")
```

Comment:
Not a big deal, but it would be better to use a StringBuilder here since it's not shared across threads. Just mentioning since I'm going to add another comment below that is more important to update, I think.

Reply:
Switched to StringBuilder